pages/common/q.md

```diff
@@ -0,0 +1,23 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file (with header for column name mapping):
+
+`q -d, -H "SELECT * from {{path/to/file}} where answer = 42"`
```

Comment:
This example seems way too complicated for the first one, see [guideline n.4](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines). Isn't there anything simpler that could be used? E.g. something closer to the "read data from stdin" example.